admin-dev/themes/new-theme/package.json

```diff
@@ -18,37 +18,46 @@
     "bootstrap-tokenfield": "^0.12.0",
     "bourbon": "^4.2.6",
     "compass": "^0.1.1",
+    "copy-webpack-plugin": "^5.1.1",
+    "clean-webpack-plugin": "^3.0.0",
     "dropzone": "^5.5.1",
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "events": "^3.0.0",
     "fos-routing": "0.0.6",
     "jets": "0.11.0",
     "jquery-ui-dist": "^1.12.1",
     "jwerty": "^0.3.2",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "lodash.escaperegexp": "^4.1.2",
     "magnific-popup": "^1.1.0",
-    "mobile-detect": "^1.4.3",
+    "mini-css-extract-plugin": "^0.8.0",
+    "mobile-detect": "^1.4.4",
     "moment": "^2.24.0",
-    "perfect-scrollbar": "^1.4.0",
+    "path": "^0.12.7",
+    "perfect-scrollbar": "^1.5.0",
     "postcss-cssnext": "^2.10.0",
     "prestakit": "^1.1.0",
     "sprintf-js": "^1.1.2",
     "tablednd": "^1.0.3",
-    "tether": "^1.4.6",
+    "tether": "^1.4.7",
     "typeahead.js": "^0.11.1",
-    "url-polyfill": "^1.1.3",
-    "vue": "^2.6.7",
+    "url-polyfill": "^1.1.7",
+    "uglify-save-license": "^0.4.1",
+    "uglifyjs-webpack-plugin": "^2.2.0",
+    "vue": "^2.6.11",
+    "vue-loader": "^15",
     "vue-resource": "^1.5.1",
     "vue-router": "^2.8.1",
     "vue-i18n": "^8.15.1",
-    "vue-template-compiler": "^2.6.7",
-    "vuex": "^2.5.0"
+    "vue-template-compiler": "^2.6.11",
+    "vuex": "^2.5.0",
+    "webpack": "^4.41",
```

Comment:
webpack should be in devDependencies :thinking:

Reply:
Then why eslint cry about it when it's in dev dep?